_includes/figure.liquid

```diff
@@ -0,0 +1,84 @@
+{% assign img_path = include.path | remove: '.jpg' | remove: '.jpeg' | remove: '.png' | remove: '.tiff' | remove: '.gif' %}
```

Comment:
I assume that this file didn't fall out of thin air. Could we add a comment with the original source please?

Reply:
figure.liquid file was sourced from the Al-Folio static website template repo:

https://github.com/alshedivat/al-folio/blob/main/_includes/figure.liquid